roles/installer/templates/cronjobs/metrics-gather.yaml.j2

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: v1
+kind: CronJob
+metadata:
+  name: {{ ansible_operator_meta.name }}-metrics-gather
+  namespace: '{{ ansible_operator_meta.namespace }}'
+  labels:
+    app.kubernetes.io/name: '{{ ansible_operator_meta.name }}-metrics-gather'
+    {{ lookup("template", "../common/templates/labels/common.yaml.j2") | indent(width=4) | trim }}
+spec:
+  schedule: "{{ _metrics_cronjob_gather_schedule }}"
+  jobTemplate:
+    spec:
```

Comment:
Should we also add some metadata here (labels/annotations) like task/web deployments ?

https://github.com/ansible/awx-operator/blob/devel/roles/installer/templates/deployments/task.yaml.j2#L22-L35